docs/send-data/opentelemetry-collector/time-reference.md

```diff
@@ -0,0 +1,102 @@
+---
+id: time-reference
+title: Timestamps, Time Zones, Time Ranges, and Date Formats for OpenTelemetry Collector
+sidebar_label: Timestamps, Time Zones, Time Ranges, and Date Formats
+description: Learn about timestamp formats recognized by the Sumo Logic OpenTelemetry (OTel) Collector and available configuration options.
+keywords:
+  - timestamp
+  - otel collector
+  - time
+  - time reference
+  - timezone
+  - time zone
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+:::info
+This page lists timestamp formats, time zones, time ranges, and date handling specific to the Sumo Logic OpenTelemetry (OTel) Collector. For general timestamp and time management across all data sources, see [Timestamps, Time Zones, Time Ranges, and Date Formats](https://github.com/docs/send-data/reference-information/time-reference).
+:::
+
+We support several options for timestamps. When collecting log data, the timestamp attached to messages is vital, both for the integrity of the data in your account, and for accurate query results.
+
+Because of the importance of timestamps, Sumo Logic indexes the timestamp of each message, making sure that data relevant to a query’s time range is returned properly in search results, which allows you to reconstruct a correct event timeline.
+
+## Timestamps
+
+The timestamp is the part of a log message that marks the time that an event occurred. During ingestion, we can detect the message timestamp, convert it to Unix epoch time (the number of milliseconds since midnight, January 1, 1970 UTC), and index it. The timestamp is parsed either using the default timestamp parsing settings, or a custom format that you specify.
+
+### OTel automated timestamp parsing
+
```

Comment:
Per @echlebek , it would be useful to include another column that describes the name of the format, e.g., ISO8601